bucket4j-spring-boot-starter/src/main/java/com/giffing/bucket4j/spring/boot/starter/config/aspect/RateLimitAspect.java

```diff
@@ -87,7 +88,7 @@ public Object processMethodsAnnotatedWithRateLimitAnnotation(ProceedingJoinPoint
         var rateLimitAnnotation = RateLimitAopUtils.getAnnotationFromMethodOrClass(method, RateLimiting.class);
 
         Method fallbackMethod = null;
-        if(rateLimitAnnotation.fallbackMethodName() != null) {
+        if(StringUtils.hasText(rateLimitAnnotation.fallbackMethodName())) {
             var fallbackMethods = Arrays.stream(method.getDeclaringClass().getMethods())
                     .filter(p -> p.getName().equals(rateLimitAnnotation.fallbackMethodName()))
                     .toList();
```

Comment:
It would be nice if you add a log(warning) if the fallback method wasn't found.  Although there is
the Bucket4jStartupCheckConfiguration to check the method name but if someone uses a custom configuration e.g. for tests there is currently no check.